### PR TITLE
Ninja

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

**Comment by marijnvdwerf** [comment]
lgtm. Is there a speed increase?

File: CMakeLists.txt [hunk 1]
```diff
@@ -150,7 +150,7 @@ else ()
     endif ()
 endif ()
 install(TARGETS "openrct2" RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
-install(TARGETS "openrct2-cli" OPTIONAL RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
```
Comment (marijnvdwerf) [inline CMakeLists.txt]: Why is this commented out?

**Comment by IntelOrca** [comment]
This is also presumably also "properly" cross platform.

File: CMakeLists.txt (hunk 1 above)
Comment (janisozaur) [reply CMakeLists.txt]: Something I have to revert, leftover from testing

**Comment by janisozaur** [comment]
> Is there a speed increase?

hard to tell, travis has been having issues lately.

**Comment by janisozaur** [comment]
It looks like it did shave off ~50s from the affected builds.

**Pull request merged**